src/Uno.UI/UI/Xaml/Controls/Button/HyperlinkButton.wasm.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Uno.UI;
+
+namespace Windows.UI.Xaml.Controls
+{
+	public partial class HyperlinkButton
+	{
+		partial void PartialInitializeProperties()
+		{
+			if (!FeatureConfiguration.ButtonBase.UseHandCursor)
+			{
+				// If we turned off pointers for button base via configuration
+				// we need to re-enable them for HyperlinkButton manually.
+				SetStyle("cursor", "pointer");
```

Comment:
I think it would be better to find a way to deactivate the application of the CSS style and let the browser fallback to its normal behavior instead.

An easy way to do that would be to introduce a _CSS attribute selector_ (https://www.w3schools.com/css/css_attribute_selectors.asp) and use an attribute on the button's DOM object to control the behavior of the CSS.

Using the same approach, it would be easy to manage the `HyperlinkButton` without any managed code.

Reply:
@carldebilly adjusted using CSS attribute selector. Now it is actually set globally for the whole window, so it avoids the need of setting attribute for each button, less interop.